pkg/scheduler/plugins/priority/priority.go

```diff
@@ -118,6 +118,45 @@ func (pp *priorityPlugin) OnSessionOpen(ssn *framework.Session) {
 		return ji.ReadyTaskNum()+ji.WaitingTaskNum() < int32(len(ji.Tasks))
 	}
 	ssn.AddJobStarvingFns(pp.Name(), jobStarvingFn)
+
+	namespaceOrderFn := func(l interface{}, r interface{}) int {
+		lv := l.(api.NamespaceName)
+		rv := r.(api.NamespaceName)
+
+		var lPriority int32
+		var rPriority int32
+		var lHighestJob string
+		var rHighestJob string
+
+		for _, info := range ssn.Jobs {
```

Comment:
Is it better to prepare the results or collect the data at the beginning of the opensession?

Reply:
Actually, after a job scheduled, it should be neglected during sorting namespaces. So every time we pushed a namespace back into `workqueue`, it should be resorted. It is time-costing, maybe I could optimize the sort order and save it in a heap.

Reply:
> Actually, after a job scheduled, it should be neglected during sorting namespaces. So every time we pushed a namespace back into `workqueue`, it should be resorted. It is time-costing, maybe I could optimize the sort order and save it in a heap.

However, jobs in ssn.jobs in allocate will not be removed. So the sorting results here are always the same, it's need to consider.

Reply:
Yes, I have not skip jobs in `Pending` status and `allocated` jobs, I need consider about it. Thank you!